### PR TITLE
Added purchases to localstorage

### PR DESCRIPTION
**Pull request by dannnylo**
Hello, 
Saving the purchases on local storage before to sync then.

Closes issue #8

### CODE REVIEW

File: src/components/SyncPurchases.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    purchaseKey: () => {
+        return `pcs::${new Date().getTime()}`
+    },
+    storePurchase: (purchase) => {
+        localStorage.setItem(SyncPurchases.purchaseKey(), JSON.stringify(purchase));
+    },
+    syncData: async () => {
+        for (var key in localStorage) {
+            if (key.indexOf('pcs::') === 0) {
+                var purchase = JSON.parse(localStorage.getItem(key))
```
Comment (gustavodiel) [inline src/components/SyncPurchases.js:21]: Do we need to parse?

Comment (dannnylo) [reply src/components/SyncPurchases.js:21]: Yes, the localStorage only supports storage strings, then I convert to string and then parse the content. 

https://github.com/izn/flipermercado-sheets/pull/9/files#diff-0294be1739eee5728f433d23be44fffcR16

Comment (dannnylo) [reply src/components/SyncPurchases.js]: https://github.com/izn/flipermercado-sheets/pull/9/files#diff-0294be1739eee5728f433d23be44fffcR16

**Comment by izn** [comment]
Thx! :heart:

**Pull request merged**